go.sum

```diff
@@ -1,4 +1,7 @@
+4d63.com/gochecknoglobals v0.0.0-20201008074935-acfc0b28355a/go.mod h1:wfdC5ZjKSPr7CybKEcgJhUOgeAQW1+7WcyK8OvUilfo=
```

Comment:
why is `go.sum` being modified?

Reply:
because we keep adding new dependencies etc, I guess